ACE/include/makeinclude/platform_clang_common.GNU

```diff
@@ -76,7 +76,7 @@ ifneq ($(DLD),)
   endif # DLD = CXX_FOR_VERSION_TEST
   # The -E option is GNU ld specific
   ifneq ($(mingw32),1)
-    GNU_LD := $(shell sh -c '$(LD_FOR_VERSION_TEST) -v 2>&1 | grep -c "GNU ld"')
+    GNU_LD := $(shell $(LD_FOR_VERSION_TEST) -v 2>&1 | grep -c 'GNU ld')
```

Comment:
At this point should we remove this conditional from the makefile?  The `else` side should work on all platforms, unless we really need the stderr content?